plugins/dell-k2/fu-dell-k2-rmm.c

```diff
@@ -25,29 +25,29 @@ fu_dell_k2_rmm_convert_version(FuDevice *device, guint64 version_raw)
 			       (guint)(version_raw >> 8) & 0xFF);
 }
 
+void
+fu_dell_k2_rmm_fix_version(FuDevice *device)
+{
+	FuDevice *parent = NULL;
+
+	/* use version given by parent */
+	parent = fu_device_get_parent(device);
+	if (parent != NULL) {
+		guint32 rmm_version;
+
+		rmm_version = fu_dell_k2_ec_get_rmm_version(parent);
+		fu_device_set_version_raw(device, GUINT32_FROM_BE(rmm_version));
```

Comment:
I don't think fu_dell_k2_ec_get_rmm_version() should be returning a BE-encoded version -- i.e. it should be doing GUINT32_FROM_BE and returning, not returning and expecting the caller to do GUINT32_FROM_BE

Reply:
fixed.